src/platform/packages/shared/kbn-profiling-utils/common/profiling.ts

```diff
@@ -377,7 +377,10 @@ export function getCalleeSource(frame: StackFrameMetadata): string {
     }
     case FrameSymbolStatus.PARTIALLY_SYMBOLYZED: {
```

Comment:
Maybe we can fix this too:

```suggestion
    case FrameSymbolStatus.PARTIALLY_SYMBOLIZED: {
```